### DISCUSSION TITLE
Tabbed graphs in Console

### DISCUSSION THREAD

**Discussion by kkenna3** [twilio-labs/paste#3967] [Office hours] [answered]
### Background

The Voice team is working on migrating the Call Insights page in Console to Console.Zen. The goal of this migration is to avoid making any drastic changes to the UI to avoid confusion by users (it's one of the top visited pages for Voice). We're trying to use as many out-of-the-box components from Paste as possible, as this page sees a lot of updates. 

We currently have a "tabbed" view for a specific section that allows a user to see an aggregated data point for a particular metric, then click into it to view the full graph (see screenshot).

### Feedback

 I wanted some advice on how to best recreate this with Paste components, whether using an existing component or primitives. The "glanceable" data is important to keep, however we're open to changing how that info is laid out. 

### Stage of design

Exploration & high fidelity

### Relevant links

[Ideas brainstormed in Figma ](https://www.figma.com/design/ajRN1lwkjUlNrsqWgFSTlE/Voice-Console-Overview?node-id=871-60581&t=XBh10t4GWN39gz6a-11)

Screenshot of existing page
![screencapture-console-twilio-us1-monitor-insights-calls-2024-07-01-15_52_31](https://github.com/twilio-labs/paste/assets/102549494/761e7756-1233-4cea-8db7-58aeec9053b2)

### Are you looking for visual design support?

- [ ] Yes

**Comment by roxanagomez** [comment]
Hi @kkenna3 Thank you for sharing! I wanted to share a past session where we discussed a very similar use case with @ceyerkes and @cdpestitschek. You can find the [recording here](https://twilio.slack.com/archives/CBEQHHDLP/p1715205915918049). We recommended using the [Visual Picker](https://paste.twilio.design/components/visual-picker#horizontal-radio) for this specific scenario. 

You can check the [explorations here](https://www.figma.com/design/KlWRgdZx0mfbPEnOB7ZuYp/Paste-Feedback?node-id=381-142) in case you wanted to try it

We can discuss in more depth during Office hours.

**Reply by krisantrobus** [reply]
Hey @kkenna3 I see you aren't signed up for our office hours. If you would like to discuss this with the team can you add an entry in our [Sign Up](https://docs.google.com/spreadsheets/d/10vD91zdwr9hjgDOV3HSGnoC0LgvYw-thmn3X_kvW0xk/edit?gid=664218049#gid=664218049) to reserve your slot.

**Comment by roxanagomez** [comment, marked as answer]
Hi @kkenna3 Thank you for coming to office hours! [Here is the recording](https://twilio.slack.com/archives/CBEQHHDLP/p1720726929333079) of today's session. 

Also, a short summary of the suggestions discussed:

There are 2 possible components that can work for this scenario:
- [Visual Picker Radio](https://paste.twilio.design/components/visual-picker#horizontal-radio)
- [Radio Button Group](https://paste.twilio.design/components/radio-button-group#guidelines)
- Both of these components have built-in labels, but since in this specific design scenario the labels need to be hidden, you may pass an `aria-label`.  aria-label takes a string description of the component which will be read aloud by ATs in place of a visible label. This is ok to use in this situation because it's very clear what the interaction is asking of the user, but in general, we recommend using visible labels for all form elements.
